Jenkinsfile.releasability

```diff
@@ -5,30 +5,23 @@ platformNames = ['osx': 'macOS', 'ios': 'iOS', 'watchos': 'watchOS', 'tvos': 'tv
 carthageXcodeVersion = '14.3.1'
 docsSwiftVersion = '5.8.1'
 
-def installationTest(platform, test, language) {
```

Comment:
This is part of the cocoa-pipeline job, so it happens after building the release but before publishing it, so I just ran it on this branch. There's also post-release testing which validates that the published release works, and that'll probably need some fiddling to fix after merging this since it's hard to validate now.